source/flux/init.js

```diff
@@ -56,6 +57,7 @@ export async function init(store: {dispatch: any => any}) {
 		store.dispatch(loadFeedbackStatus()),
 		store.dispatch(loadAcknowledgement()),
 		store.dispatch(loadFavoriteBuildings()),
+		store.dispatch(loadRecentSearches()),
```

Comment:
I can't decide if I want you to move this to componentDidMount on the course search itself. 

Historically, I've tried to limit this list to things used across the app (I realize that some of these have already violated that principle) but this is a small piece of data to load. 

I'm going to think on it while I walk home and think. Pig me in a few hours if I don't respond.

Reply:
What do you think?

Reply:
This is fine for now